test/test_pebble.py

```diff
@@ -3146,18 +3162,20 @@ def shutdown(self):
         pass
 
 
-class TestExec(unittest.TestCase):
-    def setUp(self):
-        self.client = MockClient()
-        self.time = MockTime()
-        time_patcher = unittest.mock.patch('ops.pebble.time', self.time)
+class TestExec:
+    @pytest.fixture
+    def client(self):
```

Comment:
Is the only difference with the other fixture the path? Is there some way we could unify these?